pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -8,9 +8,11 @@
     - heroes_arranged = heroes
     - hero_display_time = 6 * 1000
   - else
-    - # The order alternates person & stat.  Person alternates non-celeb and celeb.  Non-celeb is student or teacher.
-    - # We open with a celeba.
-    - # i.e. celeba, stat, non-celeb, stat, celeb, stat, non-celeb, stat, celeb, stat, etc.
+    -# 
+      The order alternates person & stat.  Person alternates non-celeb and
+      celeb.  Non-celeb is student or teacher. We open with a celeb, i.e.,
+      celeb, stat, non-celeb, stat, celeb, stat, non-celeb, stat, celeb, stat,
+      etc.
```

Comment:
Do we have a limit for # characters per line? Just stands out here because of the little leftover 'etc.' and the comment lines being shorter than the lines below.